subjects/cl-camp1/README.md

```diff
@@ -13,7 +13,7 @@ Put in a file `mastertheLS` the command line that will:
 - list the files and directories of the current directory.
 - Ignore the hidden files, the "." and the "..".
 - Separates the results with commas.
-- Order them by ascending order of creation date (the newest first).
+- Order them by ascending order of access time (the last file to be accessed first).
```

Comment:
The grammar should be fixed: these files are not to be accessed (in the future), but have been already accessed (in the past).

Reply:
i'm no telling them to access the files, i'm telling them to order by access time. 
What i wuant is for them to use the ls flag 
```
       -u     with  -lt:  sort by, and show, access time; with -l: show access time and sort by name;
              otherwise: sort by access time, newest first
```

Reply:
changed to newest first.